google-cloud-tools-plugin/ultimate/build.gradle

```diff
@@ -37,7 +37,7 @@ dependencies {
     compile files(project(':google-cloud-tools-plugin').sourceSets.main.output)
     compile files("${System.properties['java.home']}/../lib/tools.jar")
     compile ('com.google.cloud.tools:appengine-plugins-core:' + toolsLibVersion) {
-        exclude group: 'com.google.guava', module: 'guava'
```

Comment:
If my interpretation of this build file is correct, then ultimate only depends on the main module source code, not its dependencies.

But I'll add Guava back to be safe.